validation-test/stdlib/HashingPrototype.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-run-simple-swift-swift3
+// RUN: %target-run-simple-swift-swift
```

Comment:
Typo?

```diff
-// RUN: %target-run-simple-swift-swift
+// RUN: %target-run-simple-swift
```

Reply:
Can this file be removed?

It appears to be testing an old prototype, which predates the [SE-0206](https://github.com/apple/swift-evolution/blob/master/proposals/0206-hashable-enhancements.md) proposal.